mappings/net/minecraft/util/math/MathHelper.mapping

```diff
@@ -106,6 +111,11 @@ CLASS net/minecraft/class_3532 net/minecraft/util/math/MathHelper
 		ARG 0 value
 	METHOD method_15386 ceil (F)I
 	METHOD method_15387 floorMod (II)I
+	METHOD method_15388 capUnwrappedAngularChange (FFF)F
```

Comment:
What's the difference between this and `capChange`? How is an unwrapped angle treated differently than any other float?

Reply:
It's like `capChange` except it calls `subtractAngles` to subtract the angle. Using `capChange` on angles instead of this method (or `capAngularChange`) may produce uncapped or otherwise incorrect results in some cases.